Info-Orbs/src/widgets/clockWidget.cpp

```diff
@@ -65,14 +67,17 @@ void ClockWidget::update(bool force) {
 
     GlobalTime* time = GlobalTime::getInstance();
     m_hourSingle = time->getHour();
+
     m_minuteSingle = time->getMinute();
     m_secondSingle = time->getSecond();
 
     if (m_lastHourSingle != m_hourSingle || force) {
-        String currentHourPadded = String(m_hourSingle).length() == 1 ? " " + String(m_hourSingle) : String(m_hourSingle);
-
-        m_display1Didget = currentHourPadded.substring(0, 1);
-        m_display2Didget = currentHourPadded.substring(1, 2);
+        if (m_hourSingle < 10) {
+            m_display1Didget = " ";
+        } else {
+            m_display1Didget = "1";
+        }
+        m_display2Didget = m_hourSingle % 10;
```

Comment:
this will not work for 24 hours clock. First digit also can be a 2 ;)

Reply:
good catch. Fixed and tested.